requirements-dev.txt

```diff
@@ -52,4 +52,5 @@ sqlalchemy
 xarray
 xlrd
 xlsxwriter
-xlwt
\ No newline at end of file
+xlwt
+pyreadstat
```

Comment:
can you put in the correct alphabetical place

Reply:
The file wasn't sorted at all. I've sorted all lines now.